stories/time-picker/time-picker.stories.js

```diff
@@ -349,6 +361,9 @@ export const TabletMode = () => `<div style="min-height: 580px;">
 
 TabletMode.parameters = {
     docs: {
-        iframeHeight: 550
+        iframeHeight: 550,
+        description: {
+            story: 'The time picker can be displayed in tablet mode by adding `fd-time--tablet` component. '
```

Comment:
'The time picker can be displayed in tablet mode by adding the `fd-time--tablet` modifier class to the `fd-time` base class. '